dub.json

```diff
@@ -44,25 +44,19 @@
             "mainSourceFile": "source/agora/node/main.d",
             "debugVersions": [ "AgoraDebugAPIs" ],
             "excludedSourceFiles": [
-                "source/agora/cli/*",
-                "source/agora/utils/gc/*",
+                "source/agora/cli/*"
             ]
         },
         {
             "name": "traced-server",
             "targetName": "agora-traced",
             "mainSourceFile": "source/agora/node/main.d",
             "debugVersions": [ "AgoraDebugAPIs" ],
-            "preGenerateCommands": [
-                "$DUB scripts/build_tracy.d",
-            ],
-            "sourceFiles-posix": [
-                "build/TracyClient.o"
-            ],
-            "sourceFiles-windows": [
-                "build/TracyClient.obj"
-            ],
-            "excludedSourceFiles": [ "source/agora/cli/*" ]
+            "versions": [ "AgoraTracy" ],
```

Comment:
Dub gives you a `Have_Tracyd` version automatically.

Reply:
Cool, is this documented BTW?